asab/api/discovery.py

```
@@ -49,13 +52,59 @@ async def initialize(self, app):
 			await self.InternalAuth.initialize(app)
 
 
-	def _on_tick(self, msg):
+	def _on_tick600(self, msg):
```

Ruff (0.14.14)

[warning] Unused method argument: `msg`

(ARG002)